src/transformers/generation/logits_process.py

```diff
@@ -146,6 +152,10 @@ def __init__(self, min_length: int, eos_token_id: Union[int, List[int]]):
             eos_token_id = [eos_token_id]
         if not all(isinstance(i, int) for i in eos_token_id) or any(i < 0 for i in eos_token_id):
             logger.warning(f"`eos_token_id` has to be a list of positive integers, but is {eos_token_id}")
+        logger.warning(
+            f"{self.__class__.__name__} will cause recompilations for every new token if used with `torch.compile`."
+            "do not add `min_length` in generation config for efficient compilation"
```

Comment:
```suggestion
            "Do not add `min_length` in generation config for efficient compilation"
```
(this change should be propagated to the other warnings)